### PR TITLE
fix(chromedriver): get most recent version on x64 windows if multiple major versions exist

### LINKED ISSUE #472: Downloading wrong version of Chromedriver
Hi, I have one problem. When I'm trying to download the latest chromedriver:
npx webdriver-manager update --gecko=false --standalone=false --versions.chrome=85.0.4183.83
the version which is downloaded is 85.0.4183.38.
Do you have any idea, why it not download the correct version?
I/downloader - curl -oC:\TEST\Sources\Src\TestApps\node_modules\webdriver-manager\selenium/chromedriver_85.0.4183.83.zip https://chromedriver.storage.googleapis.com/85.0.4183.38/chromedriver_win32.zip

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
Attempt to fix #472 - with more context specifically at https://github.com/angular/webdriver-manager/issues/472#issuecomment-685225125

### CODE REVIEW

File: spec/binaries/chrome_xml_spec.ts [hunk 1]
```diff
@@ -65,4 +65,17 @@ describe('chrome xml reader', () => {
       done();
     });
   });
+
+  // This test case covers a bug when all the following conditions were true.
+  //  arch was 64 with multiple major versions available.
+  fit('should note get the 85.0.4183.83, 32-bit version (arch = x64)', (done) => {
```
Comment (lotz) [inline spec/binaries/chrome_xml_spec.ts:71]:
```suggestion
  it('should not get the 85.0.4183.38, 32-bit version (arch = x64)', (done) => {
```

Removing the focused `fit`, fixing a typo.

Comment (Blackbaud-BobbyEarl) [reply spec/binaries/chrome_xml_spec.ts:71]: Thank you @lotz!

**Comment by lotz** [comment]
@googlebot I fixed it.

**Comment by Blackbaud-BobbyEarl** [comment]
Hey @cnishina!  You seem to be an active contributor to this repository and were the last to make a commit.

Do you think this PR is something you could help champion through or direct to the right person?

Thanks for any help!

**Comment by sbley** [comment]
Who else would be authorized to merge this pull request? Is there a list of authorized contributers somewhere?

**Pull request merged**